starter-code/src/viking.js

```diff
@@ -1,8 +1,33 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+  this.attack = function() {
+    return this.strength;
+  };
+  this.receiveDamage = function(damage) {
+    this.health = this.health - damage;
+  };
+}
 
 // Viking
-function Viking() {}
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+function Viking(name, health, strength) {
+  Soldier.call(this, health, strength);
+  this.name = name;
+};
+Viking.prototype.receiveDamage=function (damage){
+  Soldier.__proto__.receiveDamage(damage);
```

Comment:
You could do this like this `Soldier.prototype.receiveDamage.call(this, damage);`